src/com/namelessmc/java_api/NamelessAPI.java

```diff
@@ -97,11 +97,11 @@ public void sendMinecraftGroups(final int serverId, final Map<UUID, Set<String>>
 		groups.forEach((uuid, playerGroups) -> {
 			final JsonObject playerGroupsObject = new JsonObject();
 			playerGroupsObject.add("groups", gson.toJsonTree(playerGroups));
-			groupsJson.add(uuid.toString(), playerGroupsObject);
+			groupsJson.add(uuid.toString().replace('-', ''), playerGroupsObject);
```

Comment:
There's a special function `NamelessAPI.javaUuidToWebsiteUuid()`, but I totally forgot about it :P

Reply:
So used to single quotes 😢